recipe/meta.yaml

```diff
@@ -39,6 +39,8 @@ requirements:
     - setuptools_scm >=5.0,<9.0
     - toml
     - wheel >=0.36.2
+    # seems to work without standard-distuitils
+    # - standard-distutils >3.11  # [py>=312]
```

Comment:
Let's leave this out because the build seems to work without it and it's not on conda-forge yet.

Reply:
Presumably, it's only needed for making wheels or something.